index.html

```diff
@@ -53,5 +54,237 @@ <h1 >
+                </div>
+            </div>
+        </div>
+
+        <div class="project">
+            <div class="image"></div>
+            <div class="content">
+                <h3>
+                    Multi-Post Stories <br>
+                    Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>HTML</li>
+                </ul>
+                
+                <div class="btn">
+                    <button class="greenbtn">See project</button>
+                </div>
+
+
+
+            
+            
+             
+            </div>
+
+        </div>
+        
+        <div class="project">
+            <div class="image"></div>
+            <div class="content">
+                <h3>
+                    Multi-Post Stories <br>
+                    Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>HTML</li>
+                </ul>
+                
+                <div class="btn">
+                    <button class="greenbtn">See project</button>
+                </div>
+
+
+
+            
+            
+             
+            </div>
+
+        </div>
+
+        <div class="project">
+            <div class="image"></div>
+            <div class="content">
+                <h3>
+                    Multi-Post Stories <br>
+                    Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>HTML</li>
+                </ul>
+                
+                <div class="btn">
+                    <button class="greenbtn">See project</button>
+                </div>
+
+
+
+            
+            
+             
+            </div>
+
+        </div>
+
+        <div class="project">
+            <div class="image"></div>
+            <div class="content">
+                <h3>
+                    Multi-Post Stories <br>
+                    Gain+Glory
+                </h3>
+                <ul class="languages">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                    <li>HTML</li>
+                </ul>
+                
+                <div class="btn">
+                    <button class="greenbtn">See project</button>
+                </div>
+             
+            </div>
+
+        </div>
+    </section>
+
+    <section id="about-myself">
+
+        <div class="topright">
+
+            <h1>
+                <span id="aboutme-heading">  About me</span>
+               
+            </h1>
```

Comment:
- [ ] Kindly I recommend  you to uses a header tag` hx(h2)` for the title (e.g., "About me") instead of `h1` tag for the purpose to keep HTML Hierarchy and search and optimization.